File: SrTest.Web/Scripts/Views/Home/Employee.js

this.GetEmployees = function () {
    $.ajax({
        url: "/Home/LoadData",
        type: "POST",
        dataType: "JSON",
        data: { data: document.getElementById('txtid').value },
        success: function (data) {
            var elmtTable = document.getElementById('Resultados');
            elmtTable.innerHTML = "";
            var event_data = '';
            event_data += '<table class="table" id="Resultados">'
            event_data += '<tr>';
            event_data += '<th>Id</th>';
            event_data += '<th>Name</th>';
            event_data += '<th>Contract Type Name</th>';
            event_data += '<th>Role Id</th>';
            event_data += '<th>Role Name</th>';
            event_data += '<th>Role Description</th>';
            event_data += '<th>Annual Salary</th>';

            if (data != null) {
                $.each(data, function (index, value) {


                    event_data += '<tr>';
                    event_data += '<td>' + value.Id + '</td>';
                    event_data += '<td>' + value.Name + '</td>';
                    event_data += '<td>' + value.ContractTypeName + '</td>';
                    event_data += '<td>' + value.RoleId + '</td>';
                    event_data += '<td>' + value.RoleName + '</td>';
                    event_data += '<td>' + value.RoleDescription + '</td>';
                    event_data += '<td>' + value.AnnualSalary + '</td>';
                    event_data += '<tr>';
                });
                $("#Resultados").append(event_data);
            } 
        }
    })
}

File: SrTest.Web/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace SrTest.Web.Models
{
    [Serializable]
    public class Employee
    {
        [Display(Name = "ID")]
        public int Id { get; set; }
        [Display(Name = "Name")]
        public string Name { get; set; }
        [Display(Name = "Contract Type Name")]
        public string ContractTypeName { get; set; }
        [Display(Name = "Role Id")]
        public int RoleId { get; set; }
        [Display(Name = "Role Name")]
        public string RoleName { get; set; }
        [Display(Name = "Role Description")]
        public string RoleDescription { get; set; }
        [Display(Name = "Annual Salary")]
        public string AnnualSalary { get; set; }
    }
}